model/factories.go

```diff
@@ -235,3 +235,51 @@ func NewPulseSuppressVar(value string) *Variable {
 		Value: value,
 	}
 }
+
+// Actuator factories //
+
+// shimActuatorV2 provides a typed shim for a ActuatorV2.
+type shimActuatorV2 struct {
```

Comment:
Not sure if this is what I had in mind exactly for the shim. I was thinking more along the lines of a function that made the conversion. It could look something like this, 

```
func shim(n string, p Pin, v Var) *ActuatorV2 {
    return &ActuatorV2{Name: n, Pin: string(p), Var: v}
}

func NewDeviceActuator() *Actuator {
    return shim("Device 1", PinPower1,"Power1")
}
```